conf/default-config.json

```diff
@@ -1,2 +1,4 @@
 {
+  "qldb_ledger_name": "uid2-audit-logs",
+  "qldb_table_name": "logs"
```

Comment:
add another config to turn auditing on/off, default to off until infra is set up

Reply:
Andrei did say he does want to have loki auditing, so I'm going to look into it. I'll remove the dependency for now though.